php_phongo.c

```diff
@@ -927,6 +927,42 @@ static mongoc_uri_t *php_phongo_make_uri(const char *uri_string, bson_t *options
 	return uri;
 } /* }}} */
 
+static const char *php_phongo_bson_type_to_string(bson_type_t type) /* {{{ */
+{
+	switch (type) {
+		case BSON_TYPE_EOD: return "EOD";
+		case BSON_TYPE_DOUBLE: return "double";
+		case BSON_TYPE_UTF8: return "string";
+		case BSON_TYPE_DOCUMENT: return "document";
+		case BSON_TYPE_ARRAY: return "array";
+		case BSON_TYPE_BINARY: return "Binary";
```

Comment:
Since this is really only used for URI option validation, I considered replacing these strings with our BSON class names as that is really the only way that users could provide such invalid values when constructing a Manager.

That said, it would make the output more verbose and wouldn't allow us to differentiate between things like `BSON_TYPE_CODE` and `BSON_TYPE_CODEWSCOPE`. I don't see any reason we can't refine these in the future if we like, but let me know if things are OK as-is. Note that each return value is a single term except for "Javascript with scope", "32-bit integer", and "64-bit integer".